packages/directlinespeech/src/utils/fetchJSON.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-env node */
+const fetch = require('node-fetch');
+
+// Helper function for fetching network resource as JSON
+module.exports = async function fetchJSON(url, options) {
+  const res = await fetch(url, {
+    ...options,
+    headers: {
+      ...options.headers,
+      accept: 'application/json'
+    }
+  });
+
+  if (!res.ok) {
+    throw new Error(`Failed to fetch JSON from server due to ${res.status}`);
+  }
+  const json = await res.json();
+  return json;
```

Comment:
No one use variable.

Reply:
if i do return await res.json() then eslint would complain. would you like me to suppress the eslint warning instead?